### DISCUSSION TITLE
Checkout error after pulled latest commit.

### DISCUSSION THREAD

**Discussion by lshqqytiger** [lshqqytiger/stable-diffusion-webui-directml#131] [Announcements]
There is a breaking change on latest commits [9aa5e66460cf77f5de2ce2ae6552f29c54dba883 - f158f1032fd2e977cc884328a942c6ea0a6a06c5].

```
RuntimeError: Couldn't checkout commit c9fe758757e022f05ca5a53fa8fac28889e4f1cf for K-diffusion.
```

If you have this error below after pulled latest commit, try removing `repositories\k-diffusion` and `repositories\stable-diffusion-stability-ai`.

**Comment by VeteranXT** [comment]
Where is  repositories\k-diffusion and repositories\stable-diffusion-stability-ai located at?

**Reply by lshqqytiger** [reply]
At the folder which you cloned into. The default is `stable-diffusion-webui-directml`.

**Comment by songib** [comment]
Got the same issue. already delete both folders and redownload them and got this. still working though would I merge later or wait for you guys? I didn't know how to merge it tbh. xd
https://github.com/lshqqytiger/stable-diffusion-webui-directml/issues/133#issuecomment-1566259951

**Comment by VeteranXT** [comment]
thanks.
